Sources/Views/Controller.swift

```diff
@@ -214,6 +215,12 @@ open class FastisController<Value: FastisValue>: UIViewController, JTACMonthView
      */
     public var dismissHandler: ((DismissAction) -> Void)?
 
+    /**
+     The block to execute after "Done" button will be tapped
+     */
+    @available(*, deprecated, message: "use dismissHandler: ((DismissAction) -> Void)?")
```

Comment:
It is better to replace "deprecated" with "unavailable". Deprecated is a warning and unavailable is an error.